watertap/unit_models/translators/translator_asm2d_adm1.py

```diff
@@ -191,6 +190,15 @@ def build(self):
             mutable=True,
             doc="Smoothing factor",
         )
+        self.heaviside_k = Param(
+            initialize=1e1,
+            units=pyunits.dimensionless,
+            mutable=True,
+            doc="Smooth heaviside k parameter",
+        )
+
+        def smooth_heaviside(x, k):
+            return 1 / (1 + exp(-2 * k * x))
```

Comment:
I am thinking that this belongs somewhere in watertap.core.util. Then we can import where needed. Ultimately, we probably could/should propagate this up to IDAES, but that's not our priority in the interim.